opl/http.py

```diff
@@ -1,11 +1,26 @@
 import logging
 
 import requests
+from requests.auth import HTTPBasicAuth
 
 import urllib3
-
-
-session = requests.Session()
+import os
+
+enable_new_search = os.getenv("NEW_SEARCH_ENABLED", "false").lower() == "true"
+
+if enable_new_search:
+    username = os.getenv("OPEN_SEARCH_USERNAME")
+    password = os.getenv("OPEN_SEARCH_PASSWORD")
+    session = requests.Session()
+    session.auth = HTTPBasicAuth(username, password)
+    session.verify = False
+    session.headers.update(
+        {
+            "Content-Type": "application/json",
+        }
+    )
+else:
+    session = requests.Session()
```

Comment:
Hello @sarathbrp . I do not think this is a good way to place this. I dislike this module which just provides simple requests session and some convenience functions should contain this logic